server/controllers/reviewsController.js

```diff
@@ -27,7 +27,10 @@ export default class Review {
       .then((foundRecipe) => {
         if (!foundRecipe) {
           return res.status(404)
-            .json({ message: `No recipe with id ${req.params.recipeId}` });
+            .json({
+              status: 'fail',
+              message: `No recipe with id ${req.params.recipeId}`
+            });
         }
         if (foundRecipe) {
```

Comment:
This conditional statement is not necessary. The return statement is the first block will terminate the operation if recipe is not found.

Reply:
Fixed